transport-native-unix-common-tests/src/main/java/io/netty5/channel/unix/tests/SocketTest.java

```diff
@@ -133,4 +141,104 @@ protected int level() {
     protected int optname() {
         throw new TestAbortedException("Not supported");
     }
+
+    /**
+     * This is a nested test because we cannot test {@link IovArray} without
+     * concrete native bindings.
+     */
+    @Nested
+    class IovArrayTest {
```

Comment:
This is basically a trick to get the test running as part of epoll and/or kqueue, instead of as part of the `transport-native-unix-common` or `-common-tests` modules.
We need to do this because `-common` and `-common-tests` don't actually produce any native bindings (just source artifacts), and we need to read `IOV_MAX` from `Limits` which in turn needs `LimitsStaticallyReferencedJniMethods`.